### PR TITLE
Added API definition for version 1.5.0

### PR DESCRIPTION
**Pull request by Innovative-ashwin**
Signed-off-by: Innovative-ashwin <sinha.ashwini111@gmail.com>

### CODE REVIEW

**Comment by Innovative-ashwin** [comment]
This requires testing along with FOGL_6451 , only after testing gets completed , this can be merged to develop.
Testing requires FOGL_6451 to be completed.

**Review by MarkRiddoch — DISMISSED** [review]

**Review by MarkRiddoch — DISMISSED** [review]
This looks ok to me. We do need to have an update to init.sql and upgrade and downgrade scripts for all the databases (Postgres and SQLite), this woudl be for the table that tracks the schemas.

File: C/plugins/storage/postgres/plugin.cpp [hunk 1]
```diff
@@ -39,7 +52,8 @@ static PLUGIN_INFORMATION info = {
 	"1.0.0",                  // Version
```
Comment (Innovative-ashwin) [inline C/plugins/storage/postgres/plugin.cpp:52]: @MarkRiddoch  will this change this to 1.2.0?

**Review by MarkRiddoch — CHANGES_REQUESTED** [review]

File: C/plugins/storage/postgres/connection.cpp [hunk 2]
```diff
@@ -4386,7 +4386,7 @@ int Connection::create_schema(const std::string &payload)
 									q.query = sqlIdx;
 									q.purgeOpArg = "CreatingSchema - phase 2, dropping index on tables";
 									char msg[1000] = {'\0'};
-									sprintf("Function: %s, Schema:%s, Service:%s, tableName:%s, Error in executing drop index command %s",__FUNCTION__, schema.c_str(), service.c_str(), name.c_str(), sqlIdx.c_str());
+									sprintf(msg, "Function: %s, Schema:%s, Service:%s, tableName:%s, Error in executing drop index command %s",__FUNCTION__, schema.c_str(), service.c_str(), name.c_str(), sqlIdx.c_str());
```
[the remark this answers is not in the document]
Comment (Innovative-ashwin) [reply C/plugins/storage/postgres/connection.cpp]: Fixed

File: C/plugins/storage/postgres/plugin.cpp (hunk 1 above)
Comment (Innovative-ashwin) [reply C/plugins/storage/postgres/plugin.cpp]: Fixed

**Review by MarkRiddoch — APPROVED** [review]

**Pull request merged**